dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts

```diff
@@ -110,4 +115,13 @@ export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
     // watch for hash changes, so the contract address can be simply replaced within the url
     window.addEventListener('hashchange', this.hashChangeWatcher);
   }
+
+  /**
+   * Cuts the description to a maximum of 300 characters.
+   *
+   * @param      {string}  desc    description that should be shortend
+   */
+  getShortDescription(desc: string): string {
```

Comment:
Think it would be cool to have `300` as a function param instead of it being hardcoded :) Not a must for now tho.

Reply:
Added a default parameter to this function and also extended the DbcpForm, so this can also handle configured max. characters for the description.